tutorial9.R

```diff
@@ -0,0 +1,91 @@
+# load data
+data = read.csv(file = "ponzr1.csv", header = TRUE, sep = ",")
+# custom function(params,observations)
+# unpack parameters
+#assign variables
+WT <- data[ which(data$mutation=='WT'), 2]
+M124K <- data[ which(data$mutation==' M124K'), 2]
+V456D<- data[ which(data$mutation==' V456D'), 2]
+I213N <- data[ which(data$mutation=='I213N'), 2]
+# assign parameters to expected values
+nllike<-function(p,x,y){
+  B0=p[1]
+  E=p[2]
+  sigma=exp(p[3])
+  expected=B0+E
+  nll=-sum(dnorm(x=y,mean=expected,sd=sigma,log=TRUE))
+  return(nll)
+}
+# dnorm(x,mean,sd)
+# create vector with inital guesses
+# assess max likelihood of model parameters with optim()
+initialGuess=c(1,1,1)
+fit=optim(par=initialGuess,fn=nllike,x=WT,y=V456D)
+# value=0
+
+  nllike<-function(p,x,y){

```

Comment:
Your functions are a little confusing to me because they all have the same name. Make sure to name functions differently and to have informative names.